view/src/components/produce.js

```diff
@@ -273,6 +277,66 @@ class Produce extends Component {
     return this.state.value;
   };
 
+  filteringAccordion = () => {
```

Comment:
I just reviewed this in another PR...  please merge and/or set your base branch.